src/lib/layers/editable-geojson-layer.js

```diff
@@ -95,11 +81,11 @@ export default class EditableGeoJsonLayer extends EditableLayer {
       pointRadiusMinPixels: this.props.pointRadiusMinPixels,
       pointRadiusMaxPixels: this.props.pointRadiusMaxPixels,
       lineDashJustified: this.props.lineDashJustified,
-      getLineColor: this.selectionAwareAccessor(this.props.getLineColor),
-      getFillColor: this.selectionAwareAccessor(this.props.getFillColor),
-      getRadius: this.selectionAwareAccessor(this.props.getRadius),
-      getLineWidth: this.selectionAwareAccessor(this.props.getLineWidth),
-      getLineDashArray: this.selectionAwareAccessor(this.props.getLineDashArray),
+      getLineColor: this.wrapAccessorWithContext(this.props.getLineColor),
+      getFillColor: this.wrapAccessorWithContext(this.props.getFillColor),
+      getRadius: this.wrapAccessorWithContext(this.props.getRadius),
+      getLineWidth: this.wrapAccessorWithContext(this.props.getLineWidth),
+      getLineDashArray: this.wrapAccessorWithContext(this.props.getLineDashArray),
 
       updateTriggers: {
         getLineColor: [this.props.selectedFeatureIndexes, this.props.mode],
```

Comment:
I think we can make these default to what the default accessor actually uses. For example:

`getLineColor: [this.props.selectedFeatureIndexes]`

Reply:
Separately: I wonder if we can get rid of this accessor wrapping. Can we add some feature in deck.gl that could handle it?